src/upm_stdvector.i

```diff
@@ -0,0 +1,61 @@
+%include "autobox.i"
```

Comment:
This file appears to be almost a copy of the java/std_vector.i interface file.  This is to support the use of AbstractList?  But it is only needed for Java, so should the file be called something else, like upm_java_vector.i ?